tests/integration-tests/configs/common/common.yaml

```diff
@@ -465,6 +465,12 @@ schedulers:
         instances: {{ common.INSTANCES_DEFAULT_X86 }}
         oss: ["alinux2"]
         schedulers: ["slurm"]
+  test_slurm.py::test_scontrol_reboot:
+    dimensions:
+      - regions: ["us-east-1"]
+        instances: {{ common.INSTANCES_DEFAULT_X86 }}
+        oss: ["ubuntu2004"]
```

Comment:
Is there any specific reason why you choose ubuntu2004?

Reply:
No. I just wanted to put something different from `alinux2`. Or we can stick to `alinux2` if we prefer.

Reply:
That's fine in both cases, just wanted to understand the reasoning behind the choice :-)